inference-engine/samples/common/utils/src/args_helper.cpp

```diff
@@ -70,6 +72,15 @@ void parseInputFilesArguments(std::vector<std::string> &files) {
     for (auto img = img_begin; img != img_end; ++img) {
         readInputFilesArguments(files, *img);
     }
+
+    if (files.size() < 20) {
```

Comment:
one more number, looks like it can be a global constant or #define MAX_FILES 20?